user_docs/en/userGuide.t2t

```diff
@@ -2784,11 +2784,21 @@ You can browse installed incompatible add-ons using the [incompatible add-ons ta
 + Extra Tools +[ExtraTools]
 
 ++ Log Viewer ++[LogViewer]
-The log viewer, found under Tools in the NVDA menu, allows you to view all the logging output that has occurred up until now from when you last started NVDA.
-Using NVDA+F1 will open the log viewer and display developer information about the current navigator object.
+The log viewer, found under Tools in the NVDA menu, allows you to view the logging output that has occurred since the last session of NVDA was started.
 
-Apart from reading the content, you can also Save a copy of the log file, or refresh the viewer so that it shows the most recent output since the Log viewer was opened.
-These actions are available under the viewer's Log menu.
+Apart from reading the content, you can also Save a copy of the log file, or refresh the viewer so that it loads new log output generated after the Log viewer was opened.
+These actions are available under the Log viewer's menu.
+
+The file which is displayed when you open the log viewer is saved on your computer at the file location ``%temp%\nvda.log``.
+A new log file is created each time NVDA is started.
+When this happens, the previous NVDA session's log file is moved to ``%temp%\nvda-old.log``.
+
+You can also copy a fragment of the current log file to the clipboard without opening the log viewer.
+%kc:beginInclude
+|| Name | key | Description |
+| Open log viewer | ``NVDA+f1`` | Opens the log viewer and displays developer information about the current navigator object. |
+| Copy a fragment of the log to the clipboard | ``NVDA+control+shift+f1`` | When this command is pressed once, it sets a starting point for the log content that should be captured. When pressed a second time, it copies the log content since the start point to your clipboard. |

```

Comment:
In English, I find some of the usage here to be perhaps a bit awkward.
Here is a suggestion which might smooth it.

```suggestion
| Copy a fragment of the log to the clipboard | ``NVDA+control+shift+f1`` | When pressed once, sets a starting point for the log content that should be captured. When pressed a second time, copies to the clipboard any log content which was generated in the time between the two presses. |
```

Reply:
I think it is well understandable as it is now, because we say already that you define a starting point with the first press and you copy content since that point with the second press. And it is shorter. Unless there are strong arguments for changing this, I would let it untouched.